frontend/src/components/dashboard/Dashboard.js

```diff
@@ -286,7 +290,7 @@ export default function Dashboard() {
                 </Container>
               </Route>
               <Route
-                exact path="/gallery/:id"
+                exact path="/gallery/:isMuseum/:id"
```

Comment:
I am not familiar with react route, but it seems weird that `isMuseum` is a path rather than a parameter.

Reply:
The path includes the parameters passed to that component when using a route. Is there a different method you would recommend or would you prefer something like "/gallery/museum=:isMuseum/:id" (not sure if this something I can do but I could try it out)?